install-dev/upgrade/sql/1.7.6.0.sql

```diff
@@ -15,3 +15,8 @@ CREATE TABLE `PREFIX_currency_lang` (
   ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8;
 
 /* PHP:ps_1760_copy_data_from_currency_to_currency_lang(); */;
+
+ALTER TABLE `PREFIX_admin_filter`
```

Comment:
What happens for existing filters ? If they are ignored, that's good. But in one of my attempts, the pre-existing filters made my page crash (HTTP 500)

Reply:
I tried to revert to develop, reset the database manually. Then perform two different research in two grids.
Then went back to this branch and updated the database (new update scripts removes former index and add a new one). This works fine.

Former saved filters work if grid was not modified and still use the controller/action system, new grid with filters uuid loses their former filters but can perform new ones (former filter is still in the database but it does not create any bug).